hpx/components/containers/partitioned_vector/detail/make_index_sequence.hpp

```diff
@@ -0,0 +1,113 @@
+//  Copyright (c) 2017 Antoine Tran Tan
```

Comment:
@sithell I know. The reason is that multidimensionnal view usually does not exceed dimension 3 or 4 in scientific domains. The unrolled version implemented here (for those cases) has a much faster compilation time.